doc: develop: tools: CLion: Add update note

olegznv opened the pull request:
CLion now offers a [more native approach](https://www.jetbrains.com/help/clion/zephyr.html) to working with Zephyr projects. We would like to inform users about this.

kartben commented on doc/develop/tools/clion.rst:
```diff
@@ -3,6 +3,10 @@
 CLion
 #####
 
+> **Updated:** This guide describes how to set up, build, and debug Zephyr’s sample application in CLion, using the IDE’s CMake integration. This approach is no longer optimal.
+>
+> CLion now features [native Zephyr West integration](https://www.jetbrains.com/help/clion/zephyr.html), which provides an easier and more intuitive way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but it’s still valid if you prefer to use CMake.
+
```
we use ReStructured Text, not markdown :)
you prob want something like the following (make sure to add the actual hyperling associated with "native Zephyr West integration" at the bottom of the file.

```suggestion
.. note::

   This guide describes how to set up, build, and debug Zephyr's sample application in CLion, using
   the IDE's CMake integration. This approach is no longer optimal.

   CLion now features `native Zephyr West integration`_ which provides an easier and more intuitive
   way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but is still
   valid if you prefer to use CMake.
```

olegznv replied:
Oh, got it :) Thanks for the help @kartben!

kartben commented:
@olegznv will you be able to look into squashing the commits? I understand you've made the contribution through the GitHub web UI, so you'll have to work from a local clone to do this. Let me know if I can help

olegznv commented:
@kartben I'll try to do this. Thanks!

olegznv committed:
doc: develop: tools: CLion: add update note

CLion now offers a more native approach to working with Zephyr projects.
We would like to inform users about this.

Signed-off-by: Oleg Zinovyev <oleg.zinovyev@jetbrains.com>

olegznv commented:
@kartben I've squashed commits. Please check
